pkg/commands/tls/custom/certificate/create.go

```diff
@@ -30,7 +30,7 @@ func NewCreateCommand(parent argparser.Registerer, g *global.Data) *CreateComman
 type CreateCommand struct {
 	argparser.Base
 
-	certBlob string
+	certBlob argparser.OptionalString
```

Comment:
I think this should stay as a string as the flag is marked as 'required'
```suggestion
	certBlob string
```